src/main/resources/hibernate.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do you need thid file if you have src/main/resources/META-INF/persistence.xml ?